src/DevShop/Component/DevShopControlTemplate/composer.json

```diff
@@ -145,9 +143,8 @@
             "web/sites/all/modules/contrib/{$name}/": ["type:drupal-module"],
             "web/sites/all/themes/contrib/{$name}/": ["type:drupal-theme"]
       },
-        "preserve-paths": [
-            "web/sites",
-            "web/profiles"
```

Comment:
Did you mean to delete this line 150 to preserve "web/profiles"? There's a fail in the auto test that could be related. Error message says missing the devshop.info file.